solr/modules/ltr/src/java/org/apache/solr/ltr/feature/Feature.java

```diff
@@ -124,6 +133,9 @@ public abstract FeatureWeight createWeight(
       throws IOException;
 
   public float getDefaultValue() {
```

Comment:
what is the lifecycle of Feature?
Our parameter isNullSameAsZero is query time right? so it means we could have various queries running with different values for the param, this static part doesn't convince me, let's discuss